### ISSUE TITLE
Lustre plugin crashes 

### ISSUE THREAD

**Issue by omgold** [influxdata/telegraf#7966]
### Relevant telegraf.conf:

```toml
[[inputs.lustre2]]

  ost_procfiles = [
    "/proc/fs/lustre/obdfilter/*/stats",
    "/proc/fs/lustre/osd-*/*/stats",
    "/proc/fs/lustre/obdfilter/*/job_stats",
  ]
  mds_procfiles = [
    "/proc/fs/lustre/mdt/*/md_stats",
    "/proc/fs/lustre/mdt/*/job_stats",
  ]
```

### System info:

Telegraf: tested 1.13.4 and 63dde6f73361d941f6a08b7674886dc5ef03e2e1
Lustre: 2.12.4

### Steps to reproduce:

1. enable Lustre plugin for Telegraf, include osc.*.max_pages_per_rpc in ost_procfiles
2. for Lustre, set osc.*.max_pages_per_rpc=4096
3. do large IO so you get a line in job_stats with `max:16777216`

### Expected behavior:

Plugins completes run without errors and reports all metrics.

### Actual behavior:

Plugin crashes with this output:

```
2020-08-11T06:31:00Z E! [inputs.lustre2] Error in plugin: strconv.ParseUint: parsing "sum:": invalid syntax
```

### Additional info:

The reason for the issue is, that not all 'fields' in a line of the job_stats file are separated by whitespace. If the numbers get large enough, there is no whitespace after the *:* like in `max:16777216`:

```
  write_bytes:     { samples:       62818, unit: bytes, min:    4096, max:16777216, sum:    940887313584 }
```

Will send a pull request.

**Comment by kerriwait** [comment]
@sjwang90 this issue also impacts our telegraf installation. The fix looks good. Is there a plan to add this to a new release?

**Comment by sjwang90** [comment]
@kerriwait We'll get this merged. We're currently planning our 1.17 general release for really soon so this won't make it in. But once it's merged it'll be included in master and the nightly builds. It will be included in the following patch release (1.17.1).

**Comment by Elwell** [comment]
This issue also seems to be reported upstream to the lustre developers (see [LU-13857](https://jira.whamcloud.com/browse/LU-13857) reported by @jgrund

**Comment by sjwang90** [comment]
Fixed in #7967
